src/main/resources/view/HelpWindow.fxml

```diff
@@ -19,27 +23,48 @@
         <URL value="@Fonts.css" />
         <URL value="@HelpWindow.css" />
       </stylesheets>
-
-      <HBox alignment="CENTER">
-        <children>
-          <Label fx:id="helpMessage" text="Label">
-            <HBox.margin>
-              <Insets right="5.0" />
-            </HBox.margin>
-          </Label>
-          <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL">
-            <HBox.margin>
-              <Insets left="5.0" />
-            </HBox.margin>
-          </Button>
-        </children>
-        <opaqueInsets>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </opaqueInsets>
-        <padding>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </padding>
-      </HBox>
+         <VBox minWidth="550.0">
+            <children>
+               <TableView fx:id="table" fixedCellSize="0.0" layoutY="-1.0" minHeight="500.0" minWidth="550.0" VBox.vgrow="ALWAYS">
+                 <columns>
+                   <TableColumn fx:id="action" prefWidth="111.0" reorderable="false" text="Action" />
+                   <TableColumn fx:id="format" prefWidth="232.0" reorderable="false" text="Format" />
+                     <TableColumn fx:id="example" prefWidth="229.0" reorderable="false" text="Example" />
+                 </columns>
+                  <opaqueInsets>
+                     <Insets />
+                  </opaqueInsets>
+               </TableView>
+            <HBox alignment="CENTER" layoutX="1.0" layoutY="536.0" minHeight="80.0" minWidth="550.0" style="-fx-background-color: D3DAFB50;">
+              <children>
+                <Label fx:id="helpMessage" text="Label" textFill="#424874" wrapText="true" HBox.hgrow="ALWAYS">
+                    <padding>
+                       <Insets bottom="5.0" left="10.0" right="10.0" top="5.0" />
+                    </padding>
+                </Label>
+                <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL" textFill="WHITE" wrapText="true">
+                        <font>
+                           <Font name="Open Sans SemiBold" size="13.0" />
+                        </font>
+                        <opaqueInsets>
+                           <Insets />
+                        </opaqueInsets>
+                        <HBox.margin>
+                           <Insets left="15.0" />
+                        </HBox.margin>
+                        <padding>
+                           <Insets bottom="7.0" left="12.0" right="12.0" top="7.0" />
+                        </padding></Button>
+              </children>
+              <opaqueInsets>
+                <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
+              </opaqueInsets>
+              <padding>
+                <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
+              </padding>
+            </HBox>
```

Comment:
![image](https://user-images.githubusercontent.com/62977276/136700000-aa2de446-b6ea-4d81-a58b-e04b8fb73219.png)
Is there a way to make this inherit the size of parent after resizing the help window?